### PR TITLE
My first pull request

### PR DESCRIPTION
**Pull request by fbclh**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -102,3 +102,7 @@ dist
 
 # TernJS port file
 .tern-port
+
+# Miscellaneous
+.DS_store
+test.md
```
Comment (elyor-doniyorov) [inline .gitignore:108]: Great job putting `test.md` in `.gitignore`. Please kindly create that file with the text "This file should be ignored by git" in your project directory.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="author" content="" />
+    <meta name="description" content="" />
+    <meta name="keyword" content="" />
+    <title>hello-microverse</title>
+    <link rel="stylesheet" href="index.css" type="text/css" />
+    <script src="js/index.js" async defer></script>
```
Comment (elyor-doniyorov) [inline index.html:11]: - [OPTIONAL] You can delete the unused `script `tag because Javascript is not used here

**Review by elyor-doniyorov — CHANGES_REQUESTED** [review]
Hi @fbclh, 

Good job so far!

## To highlight :clap: 
- Pull Request is made from “feature branch” into the main branch :1st_place_medal: 
- Commit messages are meaningful :100: 

There are some issues that you still need to work on to go to the next project but you are almost there!
almost-there


## Required Changes ♻️
**1. Remember to document your work [in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md)**

- Pull Request should have a descriptive title telling you what changes are inside.
- Pull Request should have a short summary describing introduced changes in more details.

**2. You are using the correct Readme template :ok_hand:  But still, the following parts are missing :point_down:** 
- Paragraph statement; Screenshots; Built with technologies; Live demo; Instructions on setting up your project locally; Authors' links; Issues page link; License link;

**3. Make sure that there are no linter errors. Use [html&css](https://github.com/microverseinc/linters-config/blob/master/html-css) linter configs**

Please check the comments under the review :point_down:  to see additional suggestions.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by fbclh** [comment]
Hello there thanks for your feedback, I have made all the changes below based on your feedback! 

* Add a meaningful descriptive title.
* Add a short summary describing the changes.
* Made all the necessary changes to the README file.
* No Linters errors were detected.
* Create test.md with the text "This file should be ignored by git".
* Delete the script tag.

**Review by ignatius22 — APPROVED** [review]
## Hi, @fbclh, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**